cmake/prerelease.lock

```diff
@@ -0,0 +1,3 @@
+-> When this file is present, an executable that identifies as a pre-release is built by default.
+-> If you want a release build instead, add `SOL_FORCE_RELEASE=On` to your `cmake` command to override the default.
+-> If you want to customize the pre-release part in the version string, you can do it via the `SOL_PRERELEASE_STRING` CMake option.
```

Comment:
Do we make use / need this feature?

Reply:
We use that for nightlies: https://github.com/ethereum/solc-bin/blob/gh-pages/.github/workflows/nightly-emscripten.yml#L53

And for CI builds we also put `ci` there: https://github.com/ethereum/solidity/blob/43f29c00da02e19ff10d43f7eb6955d627c57728/scripts/ci/build.sh#L15

I don't think we strictly need it. We could just as well have all non-release builds identify as `develop`.

This reminds, me, @nishant-sachdeva can you prepare a PR that updates [the nightly build action in solc-bin](https://github.com/ethereum/solc-bin/blob/gh-pages/.github/workflows/nightly-emscripten.yml) to use the new system? We should merge it immediately after we merge this one.